### PR TITLE
Merge SSH, SSHOptions structs.

### PR DESCRIPTION
**Pull request by maya-r**
Avoid passing arguments lots of times.
No functional change intended.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by maya-r** [comment]
fyi @rmohr

**Review by rmohr — CHANGES_REQUESTED** [review]
Thanks for tackling this. I think we can make it a little bit more future-proof by keepint the SSHOptions struct and just adding it to the new `SSH` struct. I tried to outline it in a comment. Let me know if it does not make sense for you.

File: pkg/virtctl/ssh/ssh.go [hunk 1]
```diff
@@ -40,24 +40,15 @@ const (
...
-
 func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 
 	homeDir, err := os.UserHomeDir()
 	if err != nil {
 		glog.Warningf("failed to determine user home directory: %v", err)
 	}
 
-	sshOptions := &SSHOptions{
+	c := &SSH{
```
[the remark this answers is not in the document]
Comment (maya-r) [reply pkg/virtctl/ssh/ssh.go:50]: Done

**Pull request merged**